src/Controls/PlanarControls.js

```diff
@@ -72,13 +83,13 @@ const defaultOptions = {
     zoomTravelTime: 0.2,
     zoomInFactor: 0.25,
     zoomOutFactor: 0.4,
-    maxAltitude: 12000,
+    maxAltitude: Infinity,
```

Comment:
Switching `maxAltitude` default value to `Infinity` prevents an error that could happen with orthographic camera :
If no initial camera range is given when instantiating `PlanarView`, the default computed value could exceed `maxAltitude`. This was not a problem with perspective camera, since zooming in would eventually move the camera bellow `maxAltitude`. However, zooming in with an orthographic camera doesn't move it, so it would have been impossible at any time to zoom out.

Reply:
could you add a comment in code?